packages/mobile/src/utils/translationsFetcher.ts

```diff
@@ -0,0 +1,11 @@
+import i18n from 'src/i18n'
+
+export const fetchI18nContent = (content: { [lang: string]: any }) => {
+  const language = i18n.language.toLowerCase()
+  const texts = content[language] || content[language.slice(0, 2)] || content.en
+  for (const key of Object.keys(texts)) {
+    // This is needed for newlines to show properly.
+    texts[key] = texts[key].replace(/\\n/g, '\n')
```

Comment:
This is because the Firebase console strips new lines. 
If we wrote the value programmatically, this wouldn't happen.